src/Components/SinglePortfolio/SinglePortfolioHeader.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import "../../styles.scss";
+import "./singlePortfolioHeader.scss";
 import NavigationBar from "../CommonComponents/NavigationBar";
 
 export default function SinglePortfolioHeader() {
```

Comment:
Duplicated content , can be abstracted into a component

Reply:
this has been replaced with a common component